index.html

```diff
@@ -0,0 +1,218 @@
+             both academy and industry by covering not only basic
+             and applied research,
+             but also philosophical and ethical issues
+             regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img
+                src="./assets/images/exhibition-icon.png"
+                alt="exhibition"
+                class="icon"
+              />
+              <p class="program-name">CC Exhibition</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+             both academy and industry 
+             by covering not only basic and applied research,
+             but also philosophical and ethical issues
+              regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img src="./assets/images/forum-icon.png" alt="forum" class="icon" />
+              <p class="program-name">Forum</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+             both academy and industry by covering
+              not only basic and applied research,
+             but also philosophical and ethical 
+             issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img
+                src="./assets/images/workshop-icon.png"
+                alt="workshop"
+                class="icon"
+              />
+              <p class="program-name">Workshop</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+             both academy and industry by covering
+              not only basic and applied research,
+             but also philosophical and ethical
+              issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img src="./assets/images/ignite-icon.png" alt="ignite" class="icon" />
+              <p class="program-name">CC Ignite</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+              both academy and industry by covering
+               not only basic and applied research,
+              but also philosophical and ethical
+               issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+        </ul>
+        <div class="whole-program"><a href="#">SEE THE WHOLE PROGRAM</a></div>
+        <button type="button" class="join">Join Artificial Intelligence
+          International Conference</button>
+      </section>
+
+      <section class="featured-speakers">
```

Comment:
- [ ] As requested by the previous code reviewer which is also a requirement for this project, please make sure the feature speakers section is implemented as in the design, the design has two columns for the desktop version. Please kindly fix that.